pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -24,7 +24,22 @@ import (
 	"go.uber.org/multierr"
 )
 
-func MetricsToPRW(namespace string, externalLabels map[string]string, md pdata.Metrics) (tsMap map[string]*prompb.TimeSeries, dropped int, errs error) {
+// Deprecated: [0.44.0] use `prometheusremotewrite.FromMetrics`. It does not wrap the error as `NewPermanent`.
```

Comment:
```suggestion
// Deprecated: [0.45.0] use `prometheusremotewrite.FromMetrics`. It does not wrap the error as `NewPermanent`.
```

This should count from the upcoming release, right?  It was not deprecated in `v0.44.0`.

Reply:
you are correct.